daemon/container_operations_unix.go

```diff
@@ -307,8 +327,13 @@ func (daemon *Daemon) setupConfigDir(c *container.Container) (setupErr error) {
 		if err := os.Chown(fPath, rootIDs.UID+uid, rootIDs.GID+gid); err != nil {
 			return errors.Wrap(err, "error setting ownership for config")
 		}
+	}
+
+	label.Relabel(localMountPath, c.MountLabel, false)
```

Comment:
I need to verify that this change actually works. It's the same as secrets, but could be the implementation is different further down the stack.